JobConfig/primary/RMC.fcl

```diff
@@ -0,0 +1,25 @@
+#
+# Radiative muon capture (RMC) spectrum using the Closure Approximation as measured by TRIUMF (1999)
```

Comment:
Do we need additional generator codes to distinguish RMC from the closure approximation vs tail generation to the kinematic limit?  ie between known and unknown physics?

Reply:
This is a fair question. I'm not sure what's the best choice here, it seems reasonable to have the tail measurement connected to its own process code so I'm happy to implement one if recommended.

Reply:
Lets defer this to when you have a 'RMC tail' generator.